src/parser/extract/compiled/story.ts

```diff
@@ -26,8 +26,14 @@ interface Params {
  */
 export function getStoryPropsObjectExpression(params: Params): ESTreeAST.ObjectExpression {
   const { node, filename } = params;
-  if (node.type === 'CallExpression' && node.arguments[1].type === 'ObjectExpression') {
-    return node.arguments[1];
+  if (node.type === 'CallExpression') {
+    const secondArg = node.arguments[1];
+    if (secondArg.type === 'ObjectExpression') {
+      return secondArg;
+    }
+    if (secondArg.type === 'CallExpression' && secondArg.arguments[0].type === 'ObjectExpression') {
+      return secondArg.arguments[0];
+    }
```

Comment:
I need to understand why those changes are necessary.

So far I see that it looks like there's a difference in the compiled output between some RC versions of `svelte@5.0.0-next`. Or there's something else I did possibly overlook.

Reply:
This one stems from a workaround that I had been using before story-level tags support.

I had the following workaround:
```js
// workaround: normally you should use `tags={['!autodocs']}`,
// but `@storybook/addon-svelte-csf`'s StoryProps type does not have a `tags` property,
// thus causing a type error despite working correctly.
const noDocs = () => {
	return {
		tags: ['!autodocs']
	};
};
```

Which I had used like this (first story doesn't use it, and the second one does):
```svelte
<Story
	name="Lifted Targets"
	args={{
		targets: [0x1000, 0x2000, 0x3000],
		manualTargets: [0x4000, 0x5000, 0x6000],
		value: [0x1000, 0x2000, 0x4000, 0x5000],
		liftedTargets: [0x1000, 0x4000]
	}}
	children={template}
/>
<Story
	name="Search Miscellaneous Values"
	args={{ targets: [0x1000, 0x2000], manualTargets: [0x3000, 0x4000] }}
	play={searchMiscellaneousValues}
	{...noDocs()}
	children={template}
/>
```

This results in the following compiled output. You can see that the second `Story(node_6, ...` will not be parsed with the current code:
```js
var node_5 = $.sibling(node_4, 2);

Story(node_5, {
	name: "Lifted Targets",
	args: {
		targets: [0x1000, 0x2000, 0x3000],
		manualTargets: [0x4000, 0x5000, 0x6000],
		value: [0x1000, 0x2000, 0x4000, 0x5000],
		liftedTargets: [0x1000, 0x4000]
	},
	children: template,
	parameters: {
		__svelteCsf: {
			rawCode: "<TargetPicker\n\ttargets={new SvelteSet(args.targets)}\n\tmanualTargets={new SvelteSet(args.manualTargets)}\n\tliftedTargets={new SvelteSet(args.liftedTargets)}\n\tmaxListHeight=\"800px\"\n\tbind:value\n/>"
		}
	}
});

var node_6 = $.sibling(node_5, 2);
var spread_element = $.derived(noDocs);

Story(node_6, $.spread_props({
	name: "Search Miscellaneous Values",
	args: {
		targets: [0x1000, 0x2000],
		manualTargets: [0x3000, 0x4000]
	},
	play: searchMiscellaneousValues,
	parameters: {
		__svelteCsf: {
			rawCode: "<TargetPicker\n\ttargets={new SvelteSet(args.targets)}\n\tmanualTargets={new SvelteSet(args.manualTargets)}\n\tliftedTargets={new SvelteSet(args.liftedTargets)}\n\tmaxListHeight=\"800px\"\n\tbind:value\n/>"
		}
	}
}, () => $.get(spread_element), {
	children: template
}));
```

After the (very!) recent addition of support for story-level tags, if I remove my workaround and use `tags` as usual, the current code is able to parse it correctly.
Therefore, this modification is apparently not needed for my use case.
It might be useful to keep though, in case anybody else is using weird object spread tricks, but I'm not sure if this should be supported or not.

Reply:
> This one stems from a workaround that I had been using before story-level tags support.
> 
> I had the following workaround:
> 
> ```js
> // workaround: normally you should use `tags={['!autodocs']}`,
> // but `@storybook/addon-svelte-csf`'s StoryProps type does not have a `tags` property,
> // thus causing a type error despite working correctly.
> const noDocs = () => {
> 	return {
> 		tags: ['!autodocs']
> 	};
> };
> ```

FYI, this one for sure wouldn't work correctly. This addon does parsing firstly for Storybook indexer the following `defineMeta` properties:
- `title`
- `tags`

and the following `<Story />` props:
- `name`
- `exportName`
- `tags`

Meaning that those attributes cannot be dynamic, nor have any linked references. Meaning they have to be static literal string or array of static literal strings _(for `tags`)_.